### ISSUE TITLE
Exemptions for policies in UDS Package CR

### ISSUE THREAD

**Issue by mjnagel** [defenseunicorns/uds-core#130]
### Is your feature request related to a problem? Please describe.

When I deploy my app I would like to specify a few policy exemptions because my app needs a certain privileged setup.

### Describe the solution you'd like

When I deploy my app I would like to be able to specify exemptions to policies via the UDS Package spec.

### Describe alternatives you've considered

Exemptions could be added to uds-core directly but this creates a longer feedback loop to wait on UDS Core to release the exemption.

### Additional context

While I do believe this is necessary I think it would be good for us to think through a safe way to add this support, and I think we absolutely need to provide a way to audit/display exemptions in one place so that they don't have to be discovered from across all UDS Packages + in the UDS Core code.

**Comment by zachariahmiller** [comment]
Additional feedback relating to this issue from delivery:

**Persona:** Delivery, author of zarf packages / bundles that my end user will consume

**Feature Request:** Ability to ignore certain namespaces and/or certain pods from security policies (previously enforced by Kyverno, now by Pepr in UDS Core).

**Reason:**
1. Some workloads still require things like `runAsUser: 0` , even in prod.
2. When doing rapid development sometimes it is advantageous to turn off security policies to ensure that the application will work without them, with the plan to turn them back on once the application is functional ("start from known good state")

**Severity/Urgency:** High. UDS Core is likely unusable in its current state for any workload that requires these policy exceptions due to this missing capability.

**Acceptance Criteria:**
- I don't have to rebuild any docker images and can use the "source of truth" images that are hosted in Iron Bank
- I don't have to rebuild any Zarf packages and can use the "source of truth" OCI artifacts in GHCR
- If deploying a single Zarf package, I'm able to apply the policy exceptions at deploy time using simple configuration
- If the package is part of a Bundle, I'm able to apply the policy exception at bundle build time and decide for myself whether to allow my end user to modify it or not when they go to deploy the bundle

**Nice-To-Have:**
- I don't have to write any javascript/typescript to be able to do something that I used to be able to do with Kyverno in ~4 lines of YAML.
